application/tests/features/api.oauth.feature

```diff
@@ -28,6 +28,33 @@ Feature: Testing OAuth2 endpoints
         And the "error_description" property contains "credentials"
         Then the guzzle status code should be 400
 
+    Scenario: Requesting access token with password where 2fa required without authentication code
```

Comment:
'authentication code' has a specific meaning in oauth specs. Can you change the wording here?